mesh/amqp.c

```diff
@@ -71,13 +72,19 @@ struct message {
 			size_t size;
 			uint8_t data[384];
 		} publish;
+		struct message_subscribe {
+			char topic[255];
+		} subscribe;
 		struct message_state_changed {
 			enum mesh_amqp_state state;
 		} state_changed;
 		struct message_remote_control {
 			size_t msg_len;
 			struct fd_msg msg;
 		} rc;
+		struct message_return {
+			bool result;
+		} ret;
```

Comment:
What's that for?

Reply:
This is variable to return result of method call. When everything is ok then result value is set to true and reply for dbus call is sent, otherwise DBus error is sent.

Reply:
Ok, I missed the usage.